src/sensors/odometer/interrupt/DirectionlessOdometer.hpp

```diff
@@ -38,7 +38,7 @@ class DirectionlessOdometer : public Odometer
                           InterruptCallback callback,
                           unsigned long pulsesPerMeter);
 
-    ~DirectionlessOdometer() override = default;
+    virtual ~DirectionlessOdometer() = default;
```

Comment:
:warning: **modernize-use-override** :warning:
prefer using 'override' or (rarely) 'final' instead of 'virtual'
```suggestion
    virtual ```
```suggestion
    virtual override ```